downstream/modules/eda/con-eda-http-headers.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="eda-http-headers"]
+
+= HTTP headers
+
+[role="_abstract"]
+In the context of {EDAName} and event streams, HTTP headers play a significant role because they carry the necessary context and security information about the incoming event from a third-party source (for example, GitHub, a monitoring tool, or a proprietary webhook). They include the following capabilities: 
+
+Authentication and non-repudiation::
+This is the most critical use. Headers often contain tokens, API keys, or security signatures (like an HMAC in an `X-Hub-Signature` header) that {EDAName} uses to _verify the sender's identity_ and ensure the event payload has not been tampered with. This supports non-repudiation—proof that the event came from a legitimate source.
+Event filtering and routing::
+Headers can contain identifying details like a source application ID, a tenant name, or an environment flag (`X-Environment: production`). Rulebooks can then _filter and route_ the event payload based on these headers, ensuring the event only triggers the correct automation for the corresponding environment or application.
```

Comment:
@jself-sudoku This paragraph might have to be reworded to something like this. We dont route based on the headers.
```
A customer can optionally use the header data in their rulebooks when they define the condition, if all of the required data in their conditions is in the payload then you can leave the headers empty. If there is a specific header key that a customer is using it would have to be included in the HTTP Headers field.
```

Reply:
Thanks for adding this information. I was confused because initially we asked customers not to leave the headers empty. I'm working on rewording this and will reach out in Slack soon.